### PR TITLE
Critical Metadata Review cleanup & adding Coastal Glider 559

### PR DESCRIPTION
**Pull request by snwhite**
Removing un-necessary lines from CruiseInformation, sensor_bulk_load, and eng_bulk_load following CMR.
Adding 1000 m Coastal Glider 559 (converted from Global) to eng_bulk_load, node_bulk_load, and vocab.

### CODE REVIEW

File: vocab/vocab.csv [hunk 1]
```diff
@@ -777,6 +777,13 @@ CP05MOAS-GL514-02-FLORTM000,Coastal Pioneer,Mobile Assets,Coastal Glider 514,3-W
...
 CP05MOAS-GL514-04-DOSTAM000,Coastal Pioneer,Mobile Assets,Coastal Glider 514,Dissolved Oxygen,Aanderaa,Optode 4831,0,1000
 CP05MOAS-GL514-05-PARADM000,Coastal Pioneer,Mobile Assets,Coastal Glider 514,Photosynthetically Available Radiation,Biospherical Instruments,QSP-2155,0,1000
+CP05MOAS-GL559,Coastal Pioneer,Mobile Assets,Coastal Glider 559,,,,0,1000
+CP05MOAS-GL559-00-ENG000000,Coastal Pioneer,Mobile Assets,Coastal Glider 559,Mobile Asset Controller,Teledyne Webb,G2 Slocum Gliders,0,1000
+CP05MOAS-GL559-01-ADCPAM000,Coastal Pioneer,Mobile Assets,Coastal Glider 559,Velocity Profiler (600kHz),Teledyne RDI,Explorer DVL 600 kHz,0,1000
+CP05MOAS-GL559-02-FLORTM000,Coastal Pioneer,Mobile Assets,Coastal Glider 559,3-Wavelength Fluorometer,WET Labs,ECO Puck FLBBCD-SLK,0,1000
+CP05MOAS-GL559-03-CTDGVM000,Coastal Pioneer,Mobile Assets,Coastal Glider 559,CTD,Sea-Bird,SBE Glider Payload CTD (GP-CTD),0,1000
+CP05MOAS-GL559-04-DOSTAM000,Coastal Pioneer,Mobile Assets,Coastal Glider 559,Dissolved Oxygen,Aanderaa,Optode 4831,0,1000
+CP05MOAS-GL559-05-PARADM000,Coastal Pioneer,Mobile Assets,Coastal Glider 559,Photosynthetically Available Radiation,Biospherical Instruments,QSP-2155,0,1000
```
Comment (rtravis-whoi) [inline vocab/vocab.csv:786]: looks good

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 2]
```diff
@@ -2452,8 +2452,6 @@ CGINS-WAVSSA-05321,A00657,Sensor,0,Surface Wave Spectra: WAVSS Series A,WAVSS SE
 CGINS-WAVSSA-05771,A01026,Sensor,0,Surface Wave Spectra: WAVSS Series A,WAVSS SERIES A (TRIAXYS NM),AXYS Technologies,TRIAXYS Directional Wave Sensor,TAS05771,,20140204,22660,,
 CGINS-WAVSSA-05781,A00968,Sensor,0,Surface Wave Spectra: WAVSS Series A,WAVSS SERIES A (TRIAXYS NW),AXYS Technologies,TRIAXYS Directional Wave Sensor,TAS05781,,20140812,22660,,Woods Hole Oceanographic Institution
 CGINS-WAVSSA-05791,A00969,Sensor,0,Surface Wave Spectra: WAVSS Series A,WAVSS SERIES A (TRIAXYS NW),AXYS Technologies,TRIAXYS Directional Wave Sensor,TAS05791,1.05.0002,20140812,22660,,Woods Hole Oceanographic Institution
-OL000495,OL000495,Sensor,0,Surface Wave Spectra: WAVSS Series A,WAVSS SERIES A (TRIAXYS NW),AXYS Technologies,TRIAXYS Directional Wave Sensor,CP01CNSM-00002-WAVSS,,,,TO BE DELETED and replaced by CGINS-WAVSSA-05791,
-OL000509,OL000509,Sensor,0,Surface Wave Spectra: WAVSS Series A,WAVSS SERIES A (TRIAXYS NW),AXYS Technologies,TRIAXYS Directional Wave Sensor,CP01CNSM-00003-WAVSS,,,,TO BE DELETED and replaced by CGINS-WAVSSA-05791,
```
Comment (rtravis-whoi) [inline bulk/sensor_bulk_load-AssetRecord.csv:2456]: deletions look good

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 3]
```diff
@@ -2506,47 +2504,18 @@ N00419,N00419,Sensor,0,"Camera, Uncabled","Camera, Uncabled",Kongsberg Underwate
...
-N00754,N00754,Sensor,0,Bulk Meteorology Instrument Package: METBK LGR Module,CP04OSSM-00002-METBK,Star Engineering,ASIMET,CP04OSSM-00002-METBK,,,,,TO BE DELETED and replaced by CGINS-METBKA-99999
-N00756,N00756,Sensor,0,Bulk Meteorology Instrument Package: METBK LGR Module,CP03ISSM-00002-METBK,Star Engineering,ASIMET,CP03ISSM-00002-METBK,,,,,TO BE DELETED and replaced by CGINS-METBKA-99999
-OL000006,OL000006,Sensor,0,3-Axis Motion Pack: MOPAK,MOPAK - 3-Axis Motion Pack,MicroStrain,3DM-GX3-25,GA01SUMO-00001-MOPAK,,,,,
-OL000007,OL000007,Sensor,0,Hydrogen Sensor: HYDGN,Hydrogen Sensor,RKI Instruments,M2A transmitters,GA01SUMO-00001-HYDGN1,,,,,
-OL000008,OL000008,Sensor,0,Hydrogen Sensor: HYDGN,Hydrogen Sensor,RKI Instruments,M2A transmitters,GA01SUMO-00001-HYDGN2,,,,,
-OL000092,OL000092,Sensor,0,3-Axis Motion Pack: MOPAK,MOPAK - 3-Axis Motion Pack,MicroStrain,3DM-GX3-25,GA01SUMO-00002-MOPAK,,,,,
-OL000093,OL000093,Sensor,0,Hydrogen Sensor: HYDGN,Hydrogen Sensor,RKI Instruments,M2A transmitters,GA01SUMO-00002-HYDGN1,,,,,
-OL000094,OL000094,Sensor,0,Hydrogen Sensor: HYDGN,Hydrogen Sensor,RKI Instruments,M2A transmitters,GA01SUMO-00002-HYDGN2,,,,,
```
Comment (rtravis-whoi) [inline bulk/sensor_bulk_load-AssetRecord.csv:2520]: deletions look good

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 4]
```diff
@@ -2506,47 +2504,18 @@ N00419,N00419,Sensor,0,"Camera, Uncabled","Camera, Uncabled",Kongsberg Underwate
...
-OL000238,OL000238,Sensor,0,3-Axis Motion Pack: MOPAK,MOPAK - 3-Axis Motion Pack,MicroStrain,3DM-GX3-25,CP02PMCI-00005-MOPAK,,,,,
-OL000241,OL000241,Sensor,0,3-Axis Motion Pack: MOPAK,MOPAK - 3-Axis Motion Pack,MicroStrain,3DM-GX3-25,CP02PMCI-00006-MOPAK,,,,,
-OL000244,OL000244,Sensor,0,3-Axis Motion Pack: MOPAK,MOPAK - 3-Axis Motion Pack,MicroStrain,3DM-GX3-25,CP02PMUI-00007-MOPAK,,,,,
-OL000254,OL000254,Sensor,0,3-Axis Motion Pack: MOPAK,MOPAK - 3-Axis Motion Pack,MicroStrain,3DM-GX3-25,CP03ISSM-00005-MOPAK,,,,,
-OL000255,OL000255,Sensor,0,Hydrogen Sensor: HYDGN,Hydrogen Sensor,RKI Instruments,M2A transmitters,CP03ISSM-00004-HYDGN1,,,,,
-OL000256,OL000256,Sensor,0,Hydrogen Sensor: HYDGN,Hydrogen Sensor,RKI Instruments,M2A transmitters,CP03ISSM-00004-HYDGN2,,,,,
-OL000272,OL000272,Sensor,0,3-Axis Motion Pack: MOPAK,MOPAK - 3-Axis Motion Pack,MicroStrain,3DM-GX3-25,CP04OSSM-00004-MOPAK,,,,,
-OL000273,OL000273,Sensor,0,Hydrogen Sensor: HYDGN,Hydrogen Sensor,RKI Instruments,M2A transmitters,CP04OSSM-SBD11-HYDGN,,,,,
-OL000274,OL000274,Sensor,0,Hydrogen Sensor: HYDGN,Hydrogen Sensor,RKI Instruments,M2A transmitters,CP04OSSM-SBD12-HYDGN,,,,,
```
Comment (rtravis-whoi) [inline bulk/sensor_bulk_load-AssetRecord.csv:2538]: deletions look good

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 5]
```diff
@@ -2506,47 +2504,18 @@ N00419,N00419,Sensor,0,"Camera, Uncabled","Camera, Uncabled",Kongsberg Underwate
...
 OL000300,OL000300,Sensor,0,Hydrogen Sensor: HYDGN,Hydrogen Sensor,RKI Instruments,M2A transmitters,CE02SHSM-00001-HYDGN12,,,,,
 OL000307,OL000307,Sensor,0,3-Axis Motion Pack: MOPAK,MOPAK - 3-Axis Motion Pack,MicroStrain,3DM-GX3-25,CE02SHSM-00002+MOPAK,,,,,
 OL000308,OL000308,Sensor,0,Hydrogen Sensor: HYDGN,Hydrogen Sensor,RKI Instruments,M2A transmitters,CE02SHSM-00002-HYDGN11,,,,,
 OL000309,OL000309,Sensor,0,Hydrogen Sensor: HYDGN,Hydrogen Sensor,RKI Instruments,M2A transmitters,CE02SHSM-00002-HYDGN12,,,,,
-OL000331,OL000331,Sensor,0,3-Axis Motion Pack: MOPAK,MOPAK - 3-Axis Motion Pack,MicroStrain,3DM-GX3-25,PMUI-00001-MOPAK,,,,,
-OL000335,OL000335,Sensor,0,3-Axis Motion Pack: MOPAK,MOPAK - 3-Axis Motion Pack,MicroStrain,3DM-GX3-25,PMUI-00003-MOPAK,,,,,
-OL000337,OL000337,Sensor,0,3-Axis Motion Pack: MOPAK,MOPAK - 3-Axis Motion Pack,MicroStrain,3DM-GX3-25,PMUI-00005-MOPAK,,,,,
-OL000344,OL000344,Sensor,0,3-Axis Motion Pack: MOPAK,MOPAK - 3-Axis Motion Pack,MicroStrain,3DM-GX3-25,OSPM-00002-MOPAK,,,,,
-OL000345,OL000345,Sensor,0,3-Axis Motion Pack: MOPAK,MOPAK - 3-Axis Motion Pack,MicroStrain,3DM-GX3-25,OSPM-00003-MOPAK,,,,,
```
Comment (rtravis-whoi) [inline bulk/sensor_bulk_load-AssetRecord.csv:2549]: deletions look good

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 6]
```diff
@@ -2564,30 +2533,6 @@ OL000458,OL000458,Sensor,0,Hydrogen Sensor: HYDGN,Hydrogen Sensor,RKI Instrument
...
-OL000540,OL000540,Sensor,0,Hydrogen Sensor: HYDGN,Hydrogen Sensor,RKI Instruments,M2A transmitters,CP04OSSM-SBD11-HYDGN,,,,,
-OL000541,OL000541,Sensor,0,Hydrogen Sensor: HYDGN,Hydrogen Sensor,RKI Instruments,M2A transmitters,CP04OSSM-SBD12-HYDGN,,,,,
-OL000551,OL000551,Sensor,0,3-Axis Motion Pack: MOPAK,MOPAK - 3-Axis Motion Pack,MicroStrain,3DM-GX3-25,CP04OSSM-00003-MOPAK,,,,CP04OSSM-00003-MOPAK,
-OL000552,OL000552,Sensor,0,Hydrogen Sensor: HYDGN,Hydrogen Sensor,RKI Instruments,M2A transmitters,CP04OSSM-SBD11-HYDGN,,,,CP04OSSM-SBD11-HYDGN,
-OL000553,OL000553,Sensor,0,Hydrogen Sensor: HYDGN,Hydrogen Sensor,RKI Instruments,M2A transmitters,CP04OSSM-SBD12-HYDGN,,,,CP04OSSM-SBD12-HYDGN,
-OL000563,OL000563,Sensor,0,3-Axis Motion Pack: MOPAK,MOPAK - 3-Axis Motion Pack,MicroStrain,3DM-GX3-25,CP03ISSM-00001-MOPAK,,,,CP03ISSM-00001-MOPAK,
-OL000564,OL000564,Sensor,0,Hydrogen Sensor: HYDGN,Hydrogen Sensor,RKI Instruments,M2A transmitters,CP03ISSM-00001-SBD11-HYDGN,,,,CP03ISSM-00001-SBD11-HYDGN,
-OL000565,OL000565,Sensor,0,Hydrogen Sensor: HYDGN,Hydrogen Sensor,RKI Instruments,M2A transmitters,CP03ISSM-00001-SBD12-HYDGN,,,,CP03ISSM-00001-SBD12-HYDGN,
```
Comment (rtravis-whoi) [inline bulk/sensor_bulk_load-AssetRecord.csv:2590]: deletions look good

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 7]
```diff
@@ -2596,25 +2541,3 @@ PIRSN-OVRSRA-00001,,Sensor,0,Overview Sonar,OVRSR Series A,MARUM,R2 Sonic 2022,1
...
-R00047,R00047,Sensor,0,Hydrogen Sensor: HYDGN,Hydrogen Sensor,RKI Instruments,M2A transmitters,GI01SUMO-00002-HYDGN2,,,,,
-R00056,R00056,Sensor,0,Hydrogen Sensor: HYDGN,Hydrogen Sensor,RKI Instruments,M2A transmitters,GI01SUMO-00001-HYDGN1,,,,,
-R00057,R00057,Sensor,0,Hydrogen Sensor: HYDGN,Hydrogen Sensor,RKI Instruments,M2A transmitters,GI01SUMO-00001-HYDGN2,,,,,
-R00067,R00067,Sensor,0,3-Axis Motion Pack: MOPAK,MOPAK - 3-Axis Motion Pack,MicroStrain,3DM-GX3-25,CP02PMUO-00001 MOPAK0000,,20130122,,,
-R00068,R00068,Sensor,0,3-Axis Motion Pack: MOPAK,MOPAK - 3-Axis Motion Pack,MicroStrain,3DM-GX3-25,CP02PMUO-00003 MOPAK0000,,20130122,,,
-R00069,R00069,Sensor,0,3-Axis Motion Pack: MOPAK,MOPAK - 3-Axis Motion Pack,MicroStrain,3DM-GX3-25,CP02PMUO-00004 MOPAK0000,,20130122,,,
-R00070,R00070,Sensor,0,3-Axis Motion Pack: MOPAK,MOPAK - 3-Axis Motion Pack,MicroStrain,3DM-GX3-25,CP02PMUO-00005 MOPAK0000,,20130122,,,
-R00106,,Sensor,0,3-Axis Motion Pack: MOPAK,MOPAK - 3-Axis Motion Pack,MicroStrain,3DM-GX3-25,GA01SUMO-00003-MOPAK,,,,,
-R00297,R00297,Sensor,0,3-Axis Motion Pack: MOPAK,MOPAK - 3-Axis Motion Pack,MicroStrain,3DM-GX3-25,,,,,,
```
Comment (rtravis-whoi) [inline bulk/sensor_bulk_load-AssetRecord.csv:2620]: deletions look good

**Review by rtravis-whoi — APPROVED** [review]
Everything looks good!

File: cruise/CruiseInformation.csv [hunk 8]
```diff
@@ -250,14 +202,10 @@ EK20190813,Elakha,2019-08-13T15:00:00,2019-08-13T23:00:00,Endurance ce02shsp_000
...
 AR38,Neil Armstrong,2019-09-20T14:00:00,2019-09-22T15:15:00,Pioneer glider recovery
-AR-39,Neil Armstrong,2019-09-25T16:55:00,2019-10-15T16:26:00,Pioneer 13 #TO BE DELETED
 AR39,Neil Armstrong,2019-09-25T16:55:00,2019-10-15T16:26:00,Pioneer 13
 SKQ201921S,Sikuliaq,2019-10-10T05:00:00,2019-10-24T00:00:00,Endurance 12
-AR-40,Neil Armstrong,2019-10-20T12:00:00,2019-11-06T16:00:00,Pioneer glider cp_340 recovery #TO BE DELETED
 AR40,Neil Armstrong,2019-10-20T12:00:00,2019-11-06T16:00:00,Pioneer glider cp_340 recovery
-AR-41,Neil Armstrong,2019-11-18T00:00:00,2019-11-22T23:59:59,Pioneer glider cp_379 recovery #TO BE DELETED
 AR41,Neil Armstrong,2019-11-18T00:00:00,2019-11-22T23:59:59,Pioneer glider cp_379 recovery
 EK20191121,Elakha,2019-11-21T16:14:00,2019-11-22T00:08:00,Endurance CSPP and glider operations
 TS20191212,Tenacious II,2019-12-12T15:00:00,2019-12-13T23:59:00,Pioneer 13a glider deployment
```
Comment (cdobs) [inline cruise/CruiseInformation.csv:211]: looks good

File: bulk/platform_bulk_load-AssetRecord.csv [hunk 9]
```diff
@@ -338,27 +349,31 @@ CGMGI-01SUMO-00004,,Platform,0,Global Irminger Sea Apex Surface Mooring,MOORING
...
 CGMGI-03FLMB-00003,OL000603,Platform,0,Global Irminger Sea Flanking Subsurface Mooring B,MOORING MESCALE FLANKING B SP,WHOI,GI03FLMB,GI03FLMB-00003,,,,,Woods Hole Oceanographic Institution
 CGMGI-03FLMB-00004,,Platform,0,Global Irminger Sea Flanking Subsurface Mooring B,MOORING MESCALE FLANKING B SP,WHOI,GI03FLMB,GI03FLMB-00004,,,,,Woods Hole Oceanographic Institution
 CGMGI-03FLMB-00005,,Platform,0,Global Irminger Sea Flanking Subsurface Mooring B,MOORING MESCALE FLANKING B SP,WHOI,GI03FLMB,GI03FLMB-00005,,,,,Woods Hole Oceanographic Institution
 CGMGI-03FLMB-00006,,Platform,0,Global Irminger Sea Flanking Subsurface Mooring B,MOORING MESCALE FLANKING B SP,WHOI,GI03FLMB,GI03FLMB-00006,,,,,Woods Hole Oceanographic Institution
 CGMGI-03FLMB-00007,,Platform,0,Global Irminger Sea Flanking Subsurface Mooring B,MOORING MESCALE FLANKING B SP,WHOI,GI03FLMB,GI03FLMB-00007,,,,,Woods Hole Oceanographic Institution
+CGMGI-03FLMB-00008,,Platform,0,Global Irminger Sea Flanking Subsurface Mooring B,MOORING MESCALE FLANKING B SP,WHOI,GI03FLMB,GI03FLMB-00008,,,,,Woods Hole Oceanographic Institution
```
Comment (cdobs) [inline bulk/platform_bulk_load-AssetRecord.csv:376]: all look good

File: vocab/vocab.csv (hunk 1 above)
Comment (cdobs) [inline vocab/vocab.csv:786]: looks good

File: bulk/sensor_bulk_load-AssetRecord.csv (hunk 7 above)
Comment (cdobs) [inline bulk/sensor_bulk_load-AssetRecord.csv:2620]: looks good

File: bulk/node_bulk_load-AssetRecord.csv [hunk 10]
```diff
@@ -61,6 +61,7 @@ CGVEH-GLDRCP-00387,A00585,Node,1,Coastal Glider 387,COASTAL GLIDER 1000M,Teledyn
 CGVEH-GLDRCP-00388,A00583,Node,1,Coastal Glider 388,COASTAL GLIDER 1000M,Teledyne Webb,ASSY 4335-1000,388,,20131223,186935,,Woods Hole Oceanographic Institution
 CGVEH-GLDRCP-00389,A00584,Node,1,Coastal Glider 389,COASTAL GLIDER 1000M,Teledyne Webb,ASSY 4335-1000,389,,20131223,186935,,Woods Hole Oceanographic Institution
 CGVEH-GLDRCP-00514,,Node,1,Coastal Glider 514,COASTAL GLIDER 1000M,Teledyne Webb,ASSY 4335-1000,514,,,,,Woods Hole Oceanographic Institution
+CGVEH-GLDRCP-00559,,Node,1,Coastal Profiling Glider 559,COASTAL GLIDER 1000M,Teledyne Webb,ASSY 302392-3,559,,,,,Woods Hole Oceanographic Institution
```
Comment (cdobs) [inline bulk/node_bulk_load-AssetRecord.csv:64]: Description of equipment column should be "Coastal Glider 559". This glider is being converted to a regular coastal glider and not a profiling coastal.

File: bulk/eng_bulk_load-AssetRecord.csv [hunk 11]
```diff
@@ -181,9 +177,9 @@ CGCON-GLDRCP-00387,OL000356,Sensor,1,Mobile Asset Controller for Coastal Glider,
 CGCON-GLDRCP-00388,OL000357,Sensor,1,Mobile Asset Controller for Coastal Glider,GL388 Controller,Teledyne Webb,G2 Slocum Gliders,388,,,,,Woods Hole Oceanographic Institution
 CGCON-GLDRCP-00389,OL000358,Sensor,1,Mobile Asset Controller for Coastal Glider,GL389 Controller,Teledyne Webb,G2 Slocum Gliders,389,,,,,Woods Hole Oceanographic Institution
 CGCON-GLDRCP-00514,,Sensor,1,Mobile Asset Controller for Coastal Glider,GL514 Controller,Teledyne Webb,G2 Slocum Gliders,514,,,,,Woods Hole Oceanographic Institution
+CGCON-GLDRCP-00559,,Sensor,1,Mobile Asset Controller for Coastal Glider,GL559 Controller,Teledyne Webb,G2 Slocum Gliders,559,,,,Was Global; now Coastal,Woods Hole Oceanographic Institution
```
Comment (cdobs) [inline bulk/eng_bulk_load-AssetRecord.csv:180]: Looks good. If we want to be consistent we should add the comment "Was Global; now Coastal" for 514 and 564 as well

**Review by cdobs — CHANGES_REQUESTED** [review]
**One small change in bulk/node_bulk_load-AssetRecord.csv:**
Description of equipment column should be "Coastal Glider 559". This glider is being converted to a regular coastal glider and not a profiling coastal.

**Two small changes in bulk/eng_bulk_load-AssetRecord.csv:**
If we want to be consistent we should add the comment "Was Global; now Coastal" for 514 and 564 as well (both are earlier conversions)

File: bulk/node_bulk_load-AssetRecord.csv [hunk 12]
```diff
@@ -61,6 +61,7 @@ CGVEH-GLDRCP-00387,A00585,Node,1,Coastal Glider 387,COASTAL GLIDER 1000M,Teledyn
 CGVEH-GLDRCP-00388,A00583,Node,1,Coastal Glider 388,COASTAL GLIDER 1000M,Teledyne Webb,ASSY 4335-1000,388,,20131223,186935,,Woods Hole Oceanographic Institution
 CGVEH-GLDRCP-00389,A00584,Node,1,Coastal Glider 389,COASTAL GLIDER 1000M,Teledyne Webb,ASSY 4335-1000,389,,20131223,186935,,Woods Hole Oceanographic Institution
 CGVEH-GLDRCP-00514,,Node,1,Coastal Glider 514,COASTAL GLIDER 1000M,Teledyne Webb,ASSY 4335-1000,514,,,,,Woods Hole Oceanographic Institution
+CGVEH-GLDRCP-00559,,Node,1,Coastal Glider 559,COASTAL GLIDER 1000M,Teledyne Webb,ASSY 4335-1000,559,,,,,Woods Hole Oceanographic Institution
```
Comment (cdobs) [inline bulk/node_bulk_load-AssetRecord.csv:64]: Updated description and model number look good

File: bulk/eng_bulk_load-AssetRecord.csv [hunk 13]
```diff
@@ -180,10 +176,10 @@ CGCON-GLDRCP-00380,OL000355,Sensor,1,Mobile Asset Controller for Coastal Glider,
...
 CGCON-GLDRCP-00389,OL000358,Sensor,1,Mobile Asset Controller for Coastal Glider,GL389 Controller,Teledyne Webb,G2 Slocum Gliders,389,,,,,Woods Hole Oceanographic Institution
-CGCON-GLDRCP-00514,,Sensor,1,Mobile Asset Controller for Coastal Glider,GL514 Controller,Teledyne Webb,G2 Slocum Gliders,514,,,,,Woods Hole Oceanographic Institution
-CGCON-GLDRCP-00564,,Sensor,1,Mobile Asset Controller for Coastal Glider,GL564 Controller,Teledyne Webb,G2 Slocum Gliders,564,,,,,Woods Hole Oceanographic Institution
+CGCON-GLDRCP-00514,,Sensor,1,Mobile Asset Controller for Coastal Glider,GL514 Controller,Teledyne Webb,G2 Slocum Gliders,514,,,,Was Global; now Coastal,Woods Hole Oceanographic Institution
+CGCON-GLDRCP-00559,,Sensor,1,Mobile Asset Controller for Coastal Glider,GL559 Controller,Teledyne Webb,G2 Slocum Gliders,559,,,,Was Global; now Coastal,Woods Hole Oceanographic Institution
+CGCON-GLDRCP-00564,,Sensor,1,Mobile Asset Controller for Coastal Glider,GL564 Controller,Teledyne Webb,G2 Slocum Gliders,564,,,,Was Global; now Coastal,Woods Hole Oceanographic Institution
```
Comment (cdobs) [inline bulk/eng_bulk_load-AssetRecord.csv:181]: Updated comment looks good

File: bulk/eng_bulk_load-AssetRecord.csv [hunk 14]
```diff
@@ -180,10 +176,10 @@ CGCON-GLDRCP-00380,OL000355,Sensor,1,Mobile Asset Controller for Coastal Glider,
 CGCON-GLDRCP-00387,OL000356,Sensor,1,Mobile Asset Controller for Coastal Glider,GL387 Controller,Teledyne Webb,G2 Slocum Gliders,387,,,,,Woods Hole Oceanographic Institution
 CGCON-GLDRCP-00388,OL000357,Sensor,1,Mobile Asset Controller for Coastal Glider,GL388 Controller,Teledyne Webb,G2 Slocum Gliders,388,,,,,Woods Hole Oceanographic Institution
 CGCON-GLDRCP-00389,OL000358,Sensor,1,Mobile Asset Controller for Coastal Glider,GL389 Controller,Teledyne Webb,G2 Slocum Gliders,389,,,,,Woods Hole Oceanographic Institution
-CGCON-GLDRCP-00514,,Sensor,1,Mobile Asset Controller for Coastal Glider,GL514 Controller,Teledyne Webb,G2 Slocum Gliders,514,,,,,Woods Hole Oceanographic Institution
-CGCON-GLDRCP-00564,,Sensor,1,Mobile Asset Controller for Coastal Glider,GL564 Controller,Teledyne Webb,G2 Slocum Gliders,564,,,,,Woods Hole Oceanographic Institution
+CGCON-GLDRCP-00514,,Sensor,1,Mobile Asset Controller for Coastal Glider,GL514 Controller,Teledyne Webb,G2 Slocum Gliders,514,,,,Was Global; now Coastal,Woods Hole Oceanographic Institution
```
Comment (cdobs) [inline bulk/eng_bulk_load-AssetRecord.csv:179]: Updated comment looks good

**Review by cdobs — APPROVED** [review]
Everything looks good!

**Pull request merged**